training/nvidia-bootc/Containerfile

```
@@ -157,7 +157,11 @@ fi
 RUN grep -q /usr/lib/containers/storage /etc/containers/storage.conf || \
     sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
 	/etc/containers/storage.conf && \
-	cp /run/.input/ilab /usr/local/bin/ilab
+	if [ -f /run/.input/ilab" ]; then \
```

Missing a set of quotations:
```suggestion
	if [ -f "/run/.input/ilab" ]; then \
```